multiqc/modules/rseqc/bam_stat.py

```diff
@@ -51,15 +52,20 @@ def parse_reports(self):
                 d['unique_percent'] = (float(d['mapq_gte_mapq_cut_unique']) / t)*100.0
             if 'reads_mapped_in_proper_pairs' in d:
                 d['proper_pairs_percent'] = (float(d['reads_mapped_in_proper_pairs']) / t)*100.0
-        
+
         if len(d) > 0:
             if f['s_name'] in self.bam_stat_data:
                 log.debug("Duplicate sample name found! Overwriting: {}".format(f['s_name']))
             self.add_data_source(f, section='bam_stat')
+            #Check if SE or PE
+            
+            if d['read_1'] and d['read_2'] and d['reads_mapped_in_proper_pairs'] and d['proper-paired_reads_map_to_different_chrom'] != 0:
```

Comment:
I don't think this does what you think it does. Consider this line:
```python
if a and b and c and d != 0:
```
This is directly equivalent to this:
```python
if (a is True) and (b is True) and (c is True) and (d != 0):
```
It is _not_ equivalent to this:
```python
if (a != 0) and (b != 0) and (c != 0) and (d != 0):
```

I would probably write the above like this instead:
```python
paired_end_keys = ['read_1', 'read_2', 'reads_mapped_in_proper_pairs', 'proper-paired_reads_map_to_different_chrom']
if all(d[k] != 0 for k in paired_end_keys):
```